js/scripts.js

```diff
@@ -0,0 +1,12 @@
+var femaleNames = ['Asia', 'Kasia', 'Ola', 'Jola'];
+var maleNames = ['Piotrek', 'Marek', 'Arek', 'Jarek'];
+
+var allNames = femaleNames.concat(maleNames);
+var newName = 'Marian';
+
+if (allNames.indexOf(newName) === -1) {
+    var x = allNames.push(newName);
```

Comment:
Ta deklaracja `x` jest Ci niepotrzebna, nigdzie dalej nie korzystasz z tej zmiennej. Do wyrzucenia. Zanim wyrzucisz to sprawdź sobie co ona w sobie zawiera i zastanów się dlaczego (google -> mdn `push`).